FBSimulatorControl/Management/FBSimulatorControl.m

```diff
@@ -115,4 +115,16 @@ - (BOOL)firstRunPreconditionsWithError:(NSError **)error
   return YES;
 }
 
++ (FBSimulatorPool *)poolForConfiguration:(FBSimulatorControlConfiguration *)configuration
+{
+  return [FBSimulatorPool poolWithConfiguration:configuration deviceSet:[self deviceSetForConfiguration:configuration]];
+}
+
++ (SimDeviceSet *)deviceSetForConfiguration:(FBSimulatorControlConfiguration *)configuration
```

Comment:
It's called in the constructor and I don't see the need for lazy instantation of a property that will just be immediately instantiated. It also means that the when the initializer has returned control to the caller, it is effectively immutable.